src/main/resources/swagger/api-docs.yaml

```diff
@@ -1236,6 +1236,70 @@ paths:
         '200':
           description: Successful Request
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/catalog:
+    get:
+      responses:
+        200:
+          description: Successful Request
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/WorkspaceCatalog'
+        403:
+          description: Insufficient permissions to get workspace catalog permission (must be owner or admin)
+          schema:
+            $ref: '#/definitions/ErrorReport'
```

Comment:
Does the 403 leak too much information? We don't return a 403 for regular workspace calls and treat them as 404's. Also applies to other calls as well.